cmd/cicheck.go

```diff
@@ -45,12 +46,18 @@ var cicheckCmd = &cobra.Command{
 			helper.PrintError(err)
 			os.Exit(1)
 		}
-		helper.ProcessCiCheck(fileID, riskThresholdInt)
+		timeoutMinutes, _ := cmd.Flags().GetInt("sast-timeout")
```

Comment:
```suggestion
		timeoutMinutes, _ := cmd.Flags().GetInt("timeout")
```
should be timeout instead.

Reply:
> should be timeout instead.

This has been resolved